CHANGES.rst

```diff
@@ -1,3 +1,11 @@
+11.9.1 (2022-02-21)
+===================
+
+Roman
+-----
+
+- corrected area rmap to match updates to schema [#863] 
```

Comment:
Is this entry in the right place?
I see it's under a section which was already released.

Reply:
And below there's a previous unreleased version ??